scala_proto/scala_proto.bzl

```diff
@@ -502,7 +502,8 @@ Example:
     scalapb_proto_library(
         name = "exampla_proto_scala",
         with_grpc = True,
-        deps = ["//src/proto:example_service"]
+        deps = ["//src/proto:example_service"],
+        scalac_jvm_fags = ["-Xss8M"],
```

Comment:
1. you have misspelled `flags` here.

2. I don't think we should set the default in the example, people will copy this example and setting flags should be uncommon.